backend/src/main/resources/graphql/main.graphqls

```diff
@@ -322,6 +336,7 @@ type Mutation {
     orderingProviderZipCode: String
     orderingProviderPhone: String
     deviceTypes: [String]!
+    deviceSpecimenTypes: [String]
```

Comment:
**suggestion (non-blocking):** It would be nice if the _new_ fields were correctly UUID, but I also see how it will be gross if one is a UUID and the other is a String. Dealer's choice on resolving this, but long-term we should be consistent with UUID across the board